src/screens/SignInScreen/index.tsx

```diff
@@ -160,26 +200,29 @@ class SignIn extends React.Component<Props, SignInState> {
 
     private handleSignIn = () => {
         const { email, password } = this.state;
+        const { configs: { captcha_type }, captcha_response } = this.props;
 
-        this.props.signIn({
-            email,
-            password,
-        });
+        if ((captcha_type === 'recaptcha' || captcha_type === 'geetest') && captchaLogin()) {
```

Comment:
are really we need to check both? maybe better if we do: 
``` if (captcha_type && captchaLogin())```

Reply:
There is 3rd variant `captcha_type ===  'none'`, but I guess here can be `captcha_type !== 'none`